include/nccl_ofi_scheduler.h

```diff
@@ -91,11 +91,16 @@ typedef struct nccl_net_ofi_threshold_scheduler {
 	nccl_net_ofi_scheduler_t base;
 	/* Round robin counter */
 	unsigned int rr_counter;
+	/* Multiplex round robin counter*/
+	unsigned int mux_rr_counter;
```

Comment:
We need a better name here.

Reply:
I switched to using the RoundRobin counter which does RR on each rail or multiple rails (2 QP's or 3 QP's or 4 QP's)